src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import command.AddDeadlineCommand;
+import command.AddEventCommand;
+import command.AddToDoCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.MarkDoneCommand;
+import command.TaskListCommand;
+import command.UnknownCommand;
+
+/**
+ * Represents a <code>Parser</code> object that deals with making sense of the user command.
+ */
+public class Parser {
+    private static final String COMMAND_EXIT = "bye";
+    private static final String COMMAND_MARK_DONE = "done";
+    private static final String COMMAND_DISPLAY_TASKS = "list";
+    private static final String COMMAND_ADD_TODO = "todo";
+    private static final String COMMAND_ADD_DEADLINE = "deadline";
+    private static final String COMMAND_ADD_EVENT = "event";
+    private static final String COMMAND_DELETE_TASK = "delete";
+    private static final String COMMAND_FIND = "find";
```

Comment:
This is really more of a personal preference than actual error. I would think that the commands being added are rather literal in their meaning, given that they are actual words and not magic numbers or characters.

Would it not be sufficient to use the strings themselves in the switch case statements?

Reply:
okay, I will consider the suggestion